cunumeric/deferred.py

```diff
@@ -1132,6 +1132,40 @@ def dot(self, src1, src2, stacklevel=0, callsite=None):
                 f"dot between {rhs1_array.ndim}d and {rhs2_array.ndim}d arrays"
             )
 
+    # Create array from input array and indices
+    @profile
+    @auto_convert([1])
+    @shadow_debug("choose", [1])
+    def choose(
+        self,
+        *args,
+        choices,
+        out,
+        is_tuple,
+        mode="rase",
+        stacklevel=0,
+        callsite=None,
+    ):
+        out_arr = out.base
+        index_arr = self._broadcast(out_arr.shape)
+        ch_tuple = tuple()
```

Comment:
Similarly, prefer tuple comprehension notation here `ch_tuple = tuple(... for c in args)`